lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/fraudCri/ContractTest.java

```diff
@@ -0,0 +1,857 @@
+            """;
+
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    private static final String CLIENT_ASSERTION_HEADER = "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9";
+    private static final String CLIENT_ASSERTION_BODY =
+            "eyJpc3MiOiJpcHYtY29yZSIsInN1YiI6Imlwdi1jb3JlIiwiYXVkIjoiZHVtbXlGcmF1ZENvbXBvbmVudElkIiwiZXhwIjo0MDcwOTA5NzAwLCJqdGkiOiJTY25GNGRHWHRoWllYU181azg1T2JFb1NVMDRXLUgzcWFfcDZucHYyWlVZIn0";
+    // Signature generated using JWT.io
+    private static final String CLIENT_ASSERTION_SIGNATURE =
+            "ZHj3_BoW2CJEsWA7i_Eq0tfi3HsmoHonHVmBaY7OAzOn-ZA_MYcIaRuMOCipK4_Ar1hCT1LMYveJrD0aYs1y0Q";
+
+    // We hardcode the VC headers and bodies like this so that it is easy to update them from JSON
+    // sent by the CRI team
+    private static final String VALID_VC_HEADER =
+            """
+            {
+              "typ": "JWT",
+              "alg": "ES256"
+            }
+            """;
+    // 2099-01-01 00:00:00 is 4070908800 in epoch seconds
+    private static final String VALID_EXPERIAN_FRAUD_CHECK_VC_BODY =
+            """
+                {
+                   "sub": "test-subject",
+                   "iss": "dummyFraudComponentId",
+                   "nbf": 4070908800,
+                   "exp": 4070909400,
+                   "vc": {
+                   "type": [
+                     "VerifiableCredential",
+                     "IdentityCheckCredential"
+                   ],
+                   "credentialSubject": {
+                     "name": [
+                       {
+                         "nameParts": [
+                           {
+                             "type": "GivenName",
+                             "value": "Kenneth"
+                           },
+                           {
+                             "type": "FamilyName",
+                             "value": "Decerqueira"
+                           }
+                         ]
+                       }
+                     ],
+                     "birthDate": [
+                       {
+                         "value": "1965-07-08"
+                       }
+                     ],
+                     "address": [
+                       {
+                         "streetName": "HADLEY ROAD",
+                         "addressType": "CURRENT",
+                         "postalCode": "BA2 5AA",
+                         "buildingNumber": "8",
+                         "addressLocality": "BATH"
+                       }
+                     ]
+                   },
+                   "evidence": [
+                     {
+                        "ci": [],
```

Comment:
The example VCs on this page: https://govukverify.atlassian.net/wiki/spaces/PYI/pages/3157753869/2.1.1.1.+Fraud+Verifiable+Credential+VC contain an `activityHistoryScore` and a `checkDetails` array. Is this deliberately different?

Reply:
I copied the VCs from this documentation page: 

https://govukverify.atlassian.net/wiki/spaces/PYI/pages/3178758156/2.1.1.1.1.+Example+Fraud+identity+check+credentials+inc.+edge+cases+and+errors

Not sure why they are different

Reply:
The VC may have an `identityFraudScore` and `activityHistoryScore` and I assume it would have `checkDetails` too